Spring classes announcement work in progress (#229)
SFPC/sfpc.github.io: artiststatement-spring2020.md

---
title: Artist Statement Workshop
layout: participate2
location:
contact: sfpc
---
![](/static/img/nytechzinefair2.jpg) 

# Artist Statement Workshop: A User Guide

> Date: Starting in June 1st, 2020 on Mondays. 5 sessions of weekly 3 hours classes 

Reviewer: ```suggestion
Date: Starting on June 1st, 2020, on Mondays. 5 sessions of weekly 3-hour classes
```

Much like a user guide helps non-technical readers understand a system, an artist statement introduces an unfamiliar audience to the artist’s practice and informs them about the references, questions, and thought processes behind the artist’s work. We will consider the artist statement as a practical communication tool (for grant and residency applications) as well as a more intimate map for guiding the direction and values of one’s practice. 

Students will:
- give thoughtful presentations of their own work and feedback to others
- participate in writing exercises and one-on-one editorial meetings
- read and discuss texts written by artists
- engage in group discussions about the challenges of maintaining an honest practice
- imagine alternatives to capitalist institutions as the context for our creative work

The instructor, Celine Katzman, will offer her guidance as a curator and writer who is committed to supporting artists and creative practitioners engaged with new technologies. By the end of the workshop, students will cultivate clear and compelling artist statements which will be published in a collection on the SFPC are.na.

# References

> This workshop is adapted from the Artist’s Tool Kit, a class offered during the SFPC ten week session, taught by Taeyoon Choi with teaching assistance from Celine Katzman.

Reviewer: ```suggestion
This workshop is adapted from the Artist’s Toolkit, a class offered during the 10-week SFPC session, taught by Taeyoon Choi with teaching assistance from Celine Katzman.
```

“Celine’s artist statement workshop stuck with me because it was less about producing the perfect artist statement, but instead critically reflecting on what you have done and where you want to go with your artistic practice. She was supportive in understanding my non-art background while at the same time, productively challenging my ideas.” Mark Anthony Hernandez

“Celine's mentoring motivated me to consider professional and personal values in making art. She guided me in writing an artist statement that clarified the ideas that I want to pursue with my work and that also correlates to broader concepts I want to explore within my practice.” Vivienne La

“Celine’s workshops guided us through the daunting, yet rewarding task of articulating the ideas behind our work. Our one-on-one meetings helped me contextualize my practice alongside everything else I learned at SFPC.” Danny Garfield

![](/static/img/artiststatement/fm_gold.png)

> [Family Maker](https://rkuo.net/Family-Maker), 2017-2018 by Visiting artist Ryan Kuo

Reviewer: ```suggestion
[Family Maker](https://rkuo.net/Family-Maker), 2017-2018 by visiting artist Ryan Kuo
```

Class 5 - Publishing + What’s next?: We will decide on a format and design for the class publication. We will brainstorm what artists can offer to our communities, the public, and struggling institutions during this difficult period.

# Teacher 

![](https://sfpc.io/static/img//people/celine-katzman.jpg)

[Celine Katzman](celinekatzman.com) is a Singaporean-American writer, curator, and educator based in New York. Her work is focused on supporting artists and creative practitioners who critically engage with new technologies, emphasizing social, political, and ecological responsibility. She is a NYSCA Curatorial Fellow at the Queens Museum and an instructor at the School for Poetic Computation. She has held positions at bitforms gallery, Rhizome, and other arts institutions. Celine graduated from Brown University in 2015. Her writing appears in publications such as Rhizome, The Nation, and Art in America, as well as in the New Museum's exhibition catalog, The Art Happens Here: Net Art's Archival Poetics. She has lectured at institutions including Bard College, the School of Visual Arts, and Brown University, and is a regular guest critic at residency programs.

> This class is organized by Taeyoon Choi(SFPC). 

Reviewer: ```suggestion
This class is organized by Taeyoon Choi (SFPC). 
```

# Is this the right class for me?

If this sounds like you, then YES!

- I am a practicing artist who has not yet dedicated time to writing an artist statement
- I am a practicing artist who would like to rewrite my artist statement
- I am not an artist, but I make creative projects in my spare time and I am interested in thinking about why I make them
- I am interested in engaging with creative practitioners in different fields
- I want to better articulate and explain “what I do” creatively to friends, family, and peers
- I am applying for grants and residency programs, or any program where I need to produce a statement of purpose, personal statement, or artist statement
- I want to figure out how to align my creative practice with my personal values and politics
- I have an intermediate English language reading and writing capability (ESL students are very welcome!)
 
### How much is tuition?

> $500 USD + Eventbrite Processing fee. Upon payment, your space in the class will be reserved.

Reviewer: ```suggestion
$500 USD + Eventbrite processing fee. Upon payment, your space in the class will be reserved.
```

> We are completely self-funded, which dramatically limits our ability to offer scholarships. While we price our classes reasonably to offer quality learning experiences, we understand the tuition is beyond the reach of many people. We are sensitive to the rapidly changing economy due to the COVID-19. While we can't offer scholarships like the past, we hope to support students in need. Our previous students have received support from cultural foundations, schools, current employers, or crowdfunding. We are happy to provide supporting materials as proof of acceptance. Please check out these resources for financial support in response to the COVID19 pandemic.

Reviewer: ```suggestion
We are completely self-funded, which dramatically limits our ability to offer scholarships. While we price our classes reasonably to offer quality learning experiences, we understand the tuition is beyond the reach of many people. We are sensitive to the rapidly changing economy due to covid-19. While we can't offer scholarships like the past, we hope to support students in need. Our previous students have received support from cultural foundations, schools, current employers, or crowdfunding. We are happy to provide supporting materials as proof of acceptance. Please check out these resources for financial support in response to the COVID19 pandemic.
```

- [Cloud9 Mutual Aid Doc: Financial Assistance](https://docs.google.com/document/d/1Qo_w8b6u2yXKzE7dIUmSeWqk3FFrqS1KhoCGzqcmZiQ/edit#heading=h.8jojokwzkoa7)
- [The Art of Social Distancing](https://docs.google.com/spreadsheets/d/e/2PACX-1vTt0lJMLDRlx_HsE132C3aGFa-D_rvk8rDVtkt9E7BH0jVQHrv-zD0favR98AtgTlPbNl2A5RPDH63X/pubhtml)

> We are committed to being fully transparent about how we make and spend money. In the spirit of radical openness and generosity, on which the school was founded, [we open-source our finances on Github](https://github.com/sfpc/finance-and-administration). There, you can read financial reports and download raw statements.

Reviewer: ```suggestion
We are committed to being fully transparent about how we make and spend money. In the spirit of radical openness and generosity, on which the school was founded, [we open-source our finances on GitHub](https://github.com/sfpc/finance-and-administration). There, you can read financial reports and download raw statements.
```

 

### How do I apply?

[Application](https://airtable.com/shrsWGhjsdy5r8o7j) is now open until May 20th, 2020.  
 
We will be accepting up to 15 students on a rolling basis. We will respond to your application within 5 days of submission. Rolling admissions means there are fewer and fewer slots the longer you wait, so if you’re interested in the program get your application in early!

### What if I can’t go, can I get a refund?
- Yes, we can give you 100% refund up to 10 days before class starts
- 50% refund after 10 days, until the first day of class
- No refunds can be given after the first day of class

### Accessibility 

> Our classes are conducted in spoken English with audiovisual materials such as slides, code examples and video. All classes will be held over Zoom until further notice. Please contact us if you need a special accessibility requests. In the past, we worked with American Sign Language Interpreters and Visual Interpreters, as well as accessibility and disability rights organizations to make our classes more inclusive. Our programs are rapidly changing due to the COVID-19, we are adjusting to the needs of our community. Please help us become more accessible and inclusive. info@sfpc.io 

Reviewer: ```suggestion
Our classes are conducted in spoken English with audiovisual materials such as slides, code examples and video. All classes will be held over Zoom until further notice. Please contact us if you need a special accessibility requests. In the past, we worked with American Sign Language Interpreters and Visual Interpreters, as well as accessibility and disability rights organizations to make our classes more inclusive. Our programs are rapidly changing due to covid-19 and we are adjusting to the needs of our community. Please help us become more accessible and inclusive. info@sfpc.io 
```
